spec/lita/handlers/locker_spec.rb

```diff
@@ -116,13 +121,24 @@
       expect(replies.last).to eq('(unlock) bazbat unlocked')
     end
 
+    it 'moves to the next queued person when there is one' do
+      send_command('locker resource create foobar')
+      send_command('locker label create bazbat')
+      send_command('locker label add foobar to bazbat')
+      send_command('lock bazbat', as: alice)
+      send_command('lock bazbat', as: bob)
+      send_command('unlock bazbat # with a comment', as: alice)
+      expect(replies.last).to eq('(lock) bazbat now locked by Bob (@bob)')
+    end
+
     it 'does not unlock a label when someone else locked it' do
       send_command('locker resource create foobar')
       send_command('locker label create bazbat')
       send_command('locker label add foobar to bazbat')
       send_command('lock bazbat', as: alice)
       send_command('unlock bazbat', as: bob)
-      expect(replies.last).to eq('(failed) bazbat is locked by Alice (@alice)')
+      expect(replies.last).to eq('(failed) bazbat is locked by Alice (@alice), you have been added ' \
+                                 'to the queue, type \'locker dequeue bazbat\' to be removed')
```

Comment:
This messaging seems odd: bob attempting to unlock a label he doesn't own shouldn't add him to the queue.